examples/examples_go_test.go

```diff
@@ -11,6 +12,7 @@ import (
 )
 
 func TestAccNetworkGo(t *testing.T) {
+	t.Skip("Temporarily skip while releasing new major version")
```

Comment:
Do we need a backlog item to restore this test?

Reply:
I have a cleanup to do anyway when I remove the VERSION_PREFIX